wire-ios-request-strategy/Sources/Request Strategies/User Clients/FetchingClientRequestStrategy.swift

```diff
@@ -396,19 +396,17 @@ final class UserClientByQualifiedUserIDTranscoder: IdentifierObjectSyncTranscode
     }
 
     private func markAllClientsAsUpdated(identifiers: Set<QualifiedID>) {
-        for identifier in identifiers {
-            if let user = ZMUser.fetch(
-                with: identifier.uuid,
-                domain: identifier.domain,
-                in: managedObjectContext
-            ) {
-                for client in user.clients {
+        let clients = UserClient.fetchClientsNeedingUpdateFromBackend(in: managedObjectContext)
```

Comment:
I guess you don't need the identifiers now, or do you want to log clients ids and the identifiers too so we can check on DD?

Reply:
my bad I missread